src/styles/blocks/content.scss

```
@@ -0,0 +1,44 @@
+.content {
+
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: repeat(6,$box-height);
+
+  @media (min-width: 600px) {
+    grid-template-columns: 50% 50%;
+    grid-template-rows: repeat(3,$box-height);
+  }
+
+  @media (min-width: 900px) {
+    grid-template-columns: 1fr 1fr 1fr;
```

```suggestion
    grid-template-columns: repeat(3, 1fr);
```
If you use the same value, you can use the function repeat

---

changed